src/components/index.d.ts

```diff
@@ -30,30 +30,31 @@ export declare const CheckboxInput: {
     ...rest
   }: {
     [x: string]: any
-    formChangeHandler: any
-    validator: any
+    formChangeHandler?: any
+    validator?: any
```

Comment:
On later iterations I'll deal with typing functions as I'll probably want to create interfaces and what not.